tests/mocks/configMock.ts

```diff
@@ -93,6 +93,13 @@ const registerDefaultConfig = (): void => {
       ingestionSwapUpdateJobType: 'Ingestion_Swap_Update',
       initTaskType: 'init',
     },
+    supportedIngestionSwapTypes: [
+      {
+        productType: 'RasterVectorBest',
+        productSubType: 'testProductSubType',
+      },
+    ],
+    forbiddenTypesForParallelIngestion: ['Ingestion_New', 'Ingestion_Update'],
```

Comment:
please add test to check a value from frobidden list

Reply:
what do you mean? i have a test that fails when there is a forbidden parallel job running

Reply:
https://github.com/MapColonies/ingestion-trigger/pull/6/files#diff-2bb1500fb4019c0d78e3ca5d79a125c43107ec34fa562dab5ae5420bc59fa23a:~:text=it(%27should%20throw%20conflict%20error%20when%20there%20is%20a%20conflicting%20job%20running%27%2C%20async%20()%20%3D%3E%20%7B